devops/azure-pipelines.yml

```diff
@@ -0,0 +1,83 @@
+# This pipeline does not implement the analysis template from the 
+# project-registry-devops repository because there was an issue with
+# analysis of an Angular build when checking out multiple repos.
+
+# This pipeline will trigger on commits or pull-requests made to the branch-dev
+# or main branches.  Note that static analysis with SonarCloud is only performed
+# on "long-lived" branches, which are designated as 'main' and any branch name
+# prefixed with 'branch-'.
+trigger:
+- branch-dev
+- main
+pr:
+- branch-dev
+- main
+# The sonarcloud and discord variable groups are required for their respective
+# secrets to be available in the analysis template
+variables:
+- group: project-registry-sonarcloud
+- group: project-registry-discord
```

Comment:
as elsewhere, there should be comments here to answer the question "ok but what variables do i need to create in that variable group?"

Reply:
updated per your suggestion